examples/axes_grid1/demo_colorbar_with_inset_locator.py

```diff
@@ -1,7 +1,14 @@
 """
-================================
-Demo Colorbar With Inset Locator
-================================
+=======================================================
+Controling the position of colorbars With Inset Locator
```

Comment:
```suggestion
Controlling the position of colorbars with Inset Locator
```
Not sure; should "Inset Locator" be lower-case, too?

Reply:
Shouldn‘t that be something like „using inset_axes“. The term „inset locator“ does not appear in the example at all.